tests/integration/ha_tests/helpers.py

```diff
@@ -110,7 +109,8 @@ async def scale_application(
                 status="active",
                 timeout=TIMEOUT,
                 wait_for_exact_units=desired_count,
-                raise_on_blocked=True,
+                # Can be temporarily blocked on scale down because of post upgrade checks
+                raise_on_blocked=False,
```

Comment:
i'm a bit worried we would be hiding an issue with this. I would prefer we actually tweak the idle_period instead

Reply:
I'll try to find something that works in terms of idle_period then !